iceoryx2-cal/tests/event_tests.rs

```diff
@@ -481,6 +482,134 @@ mod event {
         assert_that!(event_id, is_none);
     }
 
+    fn wait_all_collects_all_triggers<Sut: Event, F: FnMut(&mut Vec<TriggerId>, &Sut::Listener)>(
+        mut wait_call: F,
+    ) {
+        let _watchdog = Watchdog::new();
+        const REPETITIONS: usize = 8;
+        let name = generate_name();
+
+        let sut_listener = Sut::ListenerBuilder::new(&name)
+            .trigger_id_max(TriggerId::new(REPETITIONS + 1))
+            .create()
+            .unwrap();
+        let sut_notifier = Sut::NotifierBuilder::new(&name).open().unwrap();
+
+        for i in 1..REPETITIONS {
```

Comment:
Yes it has. It is called `TriggerId` on the concept layer and `EventId` on the end user API since it defines the origin of an event - trigger is the underlying building block, unknown to the user of this API level.